### PR TITLE
Internal: Editor V2 - Update dependencies [ED-10670]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,14 +110,14 @@
     "dependencies": {
         "@elementor/editor": "0.2.0",
         "@elementor/editor-app-bar": "0.2.0",
```
Comment (Nevoss) [inline package.json:112]:
```suggestion
        "@elementor/editor": "0.2.1",
        "@elementor/editor-app-bar": "0.2.1",
```

File: package.json [hunk 2]
```diff
@@ -110,14 +110,14 @@
...
-        "@elementor/editor-v1-adapters": "0.1.0",
+        "@elementor/editor-documents": "0.2.0",
+        "@elementor/editor-responsive": "0.2.0",
+        "@elementor/editor-site-navigation": "0.2.0",
+        "@elementor/editor-v1-adapters": "0.2.0",
         "@elementor/env": "0.2.0",
-        "@elementor/icons": "0.1.0",
-        "@elementor/locations": "0.1.0",
-        "@elementor/store": "0.1.0",
+        "@elementor/icons": "0.2.0",
```
Comment (Nevoss) [inline package.json:118]:
```suggestion
        "@elementor/icons": "0.2.1",
```

File: package.json [hunk 3]
```diff
@@ -110,14 +110,14 @@
...
+        "@elementor/locations": "0.2.0",
+        "@elementor/store": "0.2.0",
         "@elementor/ui": "^1.4.47",
```
Comment (Nevoss) [inline package.json:121]:
```suggestion
        "@elementor/ui": "^1.4.50",
```

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -110,14 +110,14 @@
...
         "@elementor/editor": "0.2.0",
         "@elementor/editor-app-bar": "0.2.0",
-        "@elementor/editor-documents": "0.1.0",
-        "@elementor/editor-responsive": "0.1.0",
-        "@elementor/editor-site-navigation": "0.1.0",
-        "@elementor/editor-v1-adapters": "0.1.0",
+        "@elementor/editor-documents": "0.2.0",
+        "@elementor/editor-responsive": "0.2.0",
+        "@elementor/editor-site-navigation": "0.2.0",
+        "@elementor/editor-v1-adapters": "0.2.0",
```
[the remark this answers is not in the document]
Comment (StyleShit) [reply package.json:116]: Done

File: package.json (hunk 1 above)
Comment (StyleShit) [reply package.json:112]: Done

File: package.json (hunk 2 above)
Comment (StyleShit) [reply package.json:118]: Done

File: package.json (hunk 3 above)
Comment (StyleShit) [reply package.json:121]: Done

**Review by Nevoss — APPROVED** [review]

**Pull request merged**